tests/functional/pages/surveys/census/communal/establishment-type.page.js

```diff
@@ -33,6 +33,11 @@ class EstablishmentTypePage extends MultipleChoiceWithOtherPage {
     return this
   }
 
+  setEstablishmentTypeAnswerOtherOtherText(value) {
```

Comment:
@iwootten i was getting confused thinking that the name of the function was generated from the label. apparently its from the answer value so i have updated this.